src/lib/util/dbuff.h

```diff
@@ -1479,9 +1523,16 @@ static inline ssize_t _fr_dbuff_out_memcpy_dbuff(uint8_t **out_p, fr_dbuff_t *ou
 static inline ssize_t _fr_dbuff_out_##_type(_type##_t *out, uint8_t **pos_p, fr_dbuff_t *in) \
 { \
 	fr_assert(out); \
-	FR_DBUFF_EXTEND_LOWAT_OR_RETURN(in, sizeof(_type##_t)); \
-	*out = fr_net_to_##_type((*pos_p)); \
-	return _fr_dbuff_set(pos_p, in, (*pos_p) + sizeof(_type##_t)); \
+	if (fr_dbuff_extend_lowat(NULL, in, sizeof(_type##_t)) >= sizeof(_type##_t)) { \
```

Comment:
This isn't needed because we assume that an Nbit integer will be represented by Nbits on the wire.